lib/ansible/cli/scripts/ansible_cli_stub.py

```diff
@@ -35,6 +36,8 @@
 from ansible.errors import AnsibleError, AnsibleOptionsError, AnsibleParserError
 from ansible.module_utils._text import to_text
 
+# Set the locale to the users default setting
+locale.setlocale(locale.LC_ALL, '')
```

Comment:
Hrm, if `lib/ansible/utils/display.py` relies on this, it makes its use fragile.  I think it may need to exist in `lib/ansible/utils/display.py`.  Which should be ok, because since nearly everything uses `lib/ansible/utils/display.py` it should happen early enough everywhere.